### PR TITLE
Local config remove loyalty cohort

### PR DESCRIPTION
**Pull request by kembree1**
Removed the createion of i2b2loyalty_patients view
and 
Removed the use of the i2b2loyalty_patients view in OMOPObservationPeriod
Updated OMOPObservationPeriod so that instead of using the arbitary date of 2010/1/1 if a Observation Period start or end date is null it uses the patients dob or current date instead.

### CODE REVIEW

File: MSSQL/OMOPConfig_Setup.sql [hunk 1]
```diff
@@ -0,0 +1,124 @@
+----------------------------------------------------------------------------------------------------------------------------------------
```
Comment (jklann) [inline MSSQL/OMOPConfig_Setup.sql:1]: See your other pull request for my comments here.

File: MSSQL/OMOPLoader.sql [hunk 2]
```diff
@@ -8,15 +8,11 @@
 -- FYI, now the diagnosis and procedure transforms write to four multiple target tables
```
Comment (jklann) [inline MSSQL/OMOPLoader.sql:8]: See your other pull request for my comments on this.

File: MSSQL/OMOPLoader.sql [hunk 3]
```diff
@@ -1006,11 +985,14 @@ create procedure OMOPObservationPeriod as
...
-    , case when l.patient_num is not null then 44814725 else 44814724 end enr_basis from 
-    (select patient_num, min(start_date) enr_start,max(start_date) enr_end,max(end_date) enr_end_end from i2b2visit where patient_num in (select person_id from person) group by patient_num) x
-    left outer join i2b2loyalty_patients l on l.patient_num=x.patient_num
+    select person_id
+		, ISNULL(min_start, ISNULL(dob, SYSDATETIME())) observation_period_start_date
+		, ISNULL(max_end, ISNULL(max_start, ISNULL(dob, SYSDATETIME()))) observation_period_end_date
+		, CASE when min_start IS NULL then 44814725 else 44814724 end period_type_concept_id
+	from (
+		select p.person_id, min(v.start_date) min_start, max(v.start_date) max_start, max(end_date) max_end, min(p.birth_datetime) dob from person p
+		left outer join i2b2visit v on p.person_id = v.patient_num
```
Comment (jklann) [inline MSSQL/OMOPLoader.sql:994]: Did you compare the performance of a left join to where .. in (select ...). I don't recall if I did on this one but sometimes the difference can be quite dramatic...

**Review by jklann — CHANGES_REQUESTED** [review]
The observation period change is fine except I'm curious about performance, but I made comments on your other commits in the other pull request.

File: MSSQL/OMOPLoader.sql (hunk 3 above)
Comment (jklann) [reply MSSQL/OMOPLoader.sql:994]: Did you ever compare the performance of left join to the old where.. in (select ..)? If not let's go ahead and still merge it in but could you please add a note in the code that this changed, so if we're finding it's really slow down the road we'll remember to see if that's the problem? Thanks!

Comment (kembree1) [reply MSSQL/OMOPLoader.sql:994]: I ran both select statements. In both cases it took only 1 second to return over 15K subjects observation periods. So no difference

**Pull request merged**